server/src/main/scala/cromwell/CromwellEntryPoint.scala

```diff
@@ -275,13 +279,14 @@ object CromwellEntryPoint extends GracefulStopSupport {
     })
   }
 
-  private def writeableMetadataPath(path: Option[Path]): ErrorOr[Unit] = {
+  private def validateWriteablePaths(path: Option[Path], typeOfPath: String): ErrorOr[Unit] = {
     path match {
-      case Some(p) if !metadataPathIsWriteable(p) => s"Unable to write to metadata directory: $p".invalidNel
+      case Some(p) if !pathIsWriteable(p) => s"Unable to write $typeOfPath json to directory: $p".invalidNel
```

Comment:
is `p` a directory? From the context it looks like it would be a (possibly unqualified) file name?